layers/+email/mu4e/packages.el

```diff
@@ -37,7 +37,7 @@
       (setq mu4e-completing-read-function
             (if (configuration-layer/layer-usedp 'spacemacs-ivy)
                 'ivy-completing-read
-              'helm-completing-read))
+              'helm--completing-read-default))
```

Comment:
The doc for `helm--completing-read-default` says don't use this directly but instead use `helm-comp-read`. I haven't tested this but just pointing out if you've missed.